library/gradle

```diff
@@ -1,57 +1,57 @@
 Maintainers: Keegan Witt <keeganwitt@gmail.com> (@keeganwitt)
 GitRepo: https://github.com/keeganwitt/docker-gradle.git
-GitCommit: ef27237d21e823cc0e075fff946322b5e551e8c4
+GitCommit: 605a08fb025ad34e8e9d5d2391427886e64fc9f7
 
 
 # Hotspot
 
-Tags: 6.8.3-jdk8, 6.8.3-jdk8-hotspot, 6.8-jdk8, 6.8-jdk8-hotspot, jdk8, jdk8-hotspot, 6.8.3-jdk, 6.8.3-jdk-hotspot, 6.8-jdk, 6.8-jdk-hotspot, jdk, jdk-hotspot, 6.8.3, 6.8.3-hotspot, 6.8, 6.8-hotspot, latest, hotspot
+Tags: 7.0.0-jdk8, 7.0.0-jdk8-hotspot, 7.0-jdk8, 7.0-jdk8-hotspot, jdk8, jdk8-hotspot, 7.0.0-jdk, 7.0.0-jdk-hotspot, 7.0-jdk, 7.0-jdk-hotspot, jdk, jdk-hotspot, 7.0.0, 7.0.0-hotspot, 7.0, 7.0-hotspot, latest, hotspot
 Architectures: amd64, arm32v7, arm64v8, ppc64le, s390x
 Directory: hotspot/jdk8
 
-Tags: 6.8.3-jre8, 6.8.3-jre8-hotspot, 6.8-jre8, 6.8-jre8-hotspot, jre8, jre8-hotspot, 6.8.3-jre, 6.8.3-jre-hotspot, 6.8-jre, 6.8-jre-hotspot, jre, jre-hotspot
+Tags: 7.0.0-jre8, 7.0.0-jre8-hotspot, 7.0-jre8, 7.0-jre8-hotspot, jre8, jre8-hotspot, 7.0.0-jre, 7.0.0-jre-hotspot, 7.0-jre, 7.0-jre-hotspot, jre, jre-hotspot
 Architectures: amd64, arm32v7, arm64v8, ppc64le, s390x
```

Comment:
…and [same here](https://hub.docker.com/layers/adoptopenjdk/library/adoptopenjdk/8-jre-hotspot/images/sha256-8702f57b8baae2118780264531051033b971e3c3277779cb7bf75061a813af41?context=explore).
```suggestion
Architectures: amd64, ppc64le, s390x
```